qemu/tests/cfg/boot_with_max_mem.cfg

```diff
@@ -0,0 +1,7 @@
+- boot_with_max_mem:
+    virt_test_type = qemu
+    only Linux
```

Comment:
From the manual case, it is x86_64 only.  
How about other arch ?   Does ppc64le, aarch64, s390x support max mem 8T ?

Reply:
> From the manual case, it is x86_64 only. 
yes

>How about other arch ? Does ppc64le, aarch64, s390x support max mem 8T ?
for multi-arch, please check following messages:
"For power, we had an agreement with IBM, they will cover limitations testing from their side.
For z systems, we don't have such a big machine in the pool.
For arm, it's not official support yet and we have very limited resources, QE now only focus on sanity testing. And we don't have big machines with huge mem/cpu as well."

Reply:
@yanan-fu , I already added  "only x86_64" to cfg file,please check it ,thanks